virtualization-and-containers/material/docker_slides.md

```diff
@@ -0,0 +1,252 @@
+---
+type: slide
+slideOptions:
+  transition: slide
+  width: 1400
+  height: 900
+  margin: 0.1
+---
+
+<style>
+  .reveal strong {
+  font-weight: bold;
+    color: orange;
+  }
+  .reveal p {
+    text-align: left;
+  }
+  .reveal section h1 {
+    color: orange;
+  }
+  .reveal section h2 {
+    color: orange;
+  }
+</style>
+
+# Docker
+
+<img src="https://www.docker.com/sites/default/files/d8/2019-07/Moby-logo.png" width=20%; style="margin-left:auto; margin-right:auto; padding-top: 25px; padding-bottom: 25px">
+
+[https://www.docker.com/company/newsroom/media-resources](https://www.docker.com/company/newsroom/media-resources)
+
+---
+
+## What is Docker? 1/2
+
+- Docker, Inc.
+  - Company promoting Docker
+- Docker Desktop
+  > Developer productivity tools and a local Kubernetes environment.
+- Docker Engine
+  > Docker Engine is an open source containerization technology for building and containerizing your applications.
+
+---
+
+## What is Docker? 2/2
+
+- Docker Hub
+  > Cloud-based application registry and development team collaboration services.
+- Docker Compose
+  > Compose is a tool for defining and running multi-container Docker applications.
+- According to documentation
+  > Docker is an open platform for developing, shipping, and running applications.
+- We use definition from documentation.
+
+---
+
+## Typical Docker Applications
+
+- Applications as Microservices
+- Containers for consistent development environment
+- Containers for consistent testing environment
+- Portable format for sharing applications
+- Avoid tedious installation procedures by providing Docker container ([FEniCS](https://fenicsproject.org/download/), [GitLab](https://docs.gitlab.com/ee/install/docker.html)...)
+
+---
+
+## Building Blocks 1/2
+
+- Docker daemon `dockerd`
+  - Controlling instance of containers and reacts to API requests
+  - Server process
+- Docker client
+  - User interface/tools to interact with, create, manage containers etc. via daemon
```

Comment:
Does our audience know what you mean with "via daemon"?

Reply:
Probably not, but that they should learn. It will be more clear two slides later. I want to introduce the main terms here and then describe it in more detail when we come to the Docker image.